tests/test_cmdline.py

```diff
@@ -347,6 +349,108 @@ def test_combine(self):
                        os.path.join(example_json_dir, "different_sorts.json")],
                       expected_output_file="result.json")
 
+    def test_set_sync_marker(self):
+        test_script = """
+from viztracer import get_tracer
+
+if get_tracer() is not None:
+    get_tracer().set_sync_marker()
+
+def fib(n):
+    if n < 2:
+        return 1
+    return fib(n-1) + fib(n-2)
+fib(5)
+
+if get_tracer() is not None:
+    get_tracer().set_sync_marker()
+
+"""
+
+        def expect_sync_marker(data):
+            self.assertIsNotNone(data['viztracer_metadata'].get('sync_marker'))
+            self.assertGreater(data['viztracer_metadata'].get('sync_marker'), 0)
+
+        self.template(
+            [sys.executable, "-m", "viztracer", "--ignore_frozen", "-o", "result.json", "cmdline_test.py"],
+            expected_output_file="result.json",
+            expected_stderr="Synchronization marker already set",
+            script=test_script,
+            check_func=expect_sync_marker,
+        )
+
+    @package_matrix(["~orjson", "orjson"])
```

Comment:
`package_matrix` actually costs plenty of time and it should only be enabled when necessary (for meaningful coverage). I don't think this specific feature requires the coverage for with and without `orjson`. Maybe just remove this?